plugin/inlay_hint.py

```diff
@@ -97,7 +97,8 @@ def inlay_hint_to_phantom(view: sublime.View, inlay_hint: InlayHint, session: Se
 
 
 def get_inlay_hint_html(view: sublime.View, inlay_hint: InlayHint, session: Session, phantom_uuid: str) -> str:
-    label = format_inlay_hint_label(inlay_hint, session, phantom_uuid)
+    truncate_limit = userprefs().inlay_hints_truncate_limit
+    label = format_inlay_hint_label(inlay_hint, session, phantom_uuid, truncate_limit)
```

Comment:
Seems like it would be fine to read the pref in `format_inlay_hint_label` directly since we don't intend for `format_inlay_hint_label` to get called with any other value than the one from the pref.

Reply:
Yup, done in: [08b9795](https://github.com/sublimelsp/LSP/pull/2514/commits/08b9795cdb829d43d6b4d6716f75a3eb38ce22ca)